lib/ClientConnection.cc

```diff
@@ -679,11 +679,32 @@ void ClientConnection::processIncomingBuffer() {
         if (incomingCmd.type() == BaseCommand::MESSAGE) {
             // Parse message metadata and extract payload
             proto::MessageMetadata msgMetadata;
+            proto::BrokerEntryMetadata brokerEntryMetadata;
 
             // read checksum
             uint32_t remainingBytes = frameSize - (cmdSize + 4);
             bool isChecksumValid = verifyChecksum(incomingBuffer_, remainingBytes, incomingCmd);
 
+            auto readerIndex = incomingBuffer_.readerIndex();
+            if (incomingBuffer_.readUnsignedShort() == Commands::magicBrokerEntryMetadata) {
+                // broker entry metadata is present
+                uint32_t brokerEntryMetadataSize = incomingBuffer_.readUnsignedInt();
+                if (!brokerEntryMetadata.ParseFromArray(incomingBuffer_.data(), brokerEntryMetadataSize)) {
+                    LOG_ERROR(cnxString_ << "[consumer id " << incomingCmd.message().consumer_id()  //
+                                         << ", message ledger id "
+                                         << incomingCmd.message().message_id().ledgerid()  //
+                                         << ", entry id " << incomingCmd.message().message_id().entryid()
+                                         << "] Error parsing broker entry metadata");
```

Comment:
```suggestion
                    LOG_ERROR(cnxString_ << "[consumer id " << incomingCmd.message().consumer_id()  
                                         << ", message ledger id "
                                         << incomingCmd.message().message_id().ledgerid()  
                                         << ", entry id " << incomingCmd.message().message_id().entryid()
                                         << "] Error parsing broker entry metadata");
```

Reply:
The comment here might be added to align the `<<` symbols, otherwise it wouldn't pass the clang-format check.

Reply:
@RobertIndie @BewareMyPower Fixed